docs/usage.md

```diff
@@ -201,7 +201,7 @@ const sequelize = new Sequelize('sqlite:relativePath/dbname.db')
 
 ### PostgreSQL
 
-The library for PostgreSQL is`pg@~3.6.0` You'll just need to define the dialect:
+The library for PostgreSQL is`pg@~6.1.0` You'll just need to define the dialect:
```

Comment:
`7` doesnt match `~6.1.0`. The whole range of supported versions should be documented.